e2e/actions/l2_challenger_test.go

```diff
@@ -66,7 +66,94 @@ interaction:
 	require.Equal(rt.t, chal.StatusApproved, status)
 }
 
-func TestChallengerProvingTimeout(t *testing.T) {
+func TestChallengeAsserterBisectTimeout(t *testing.T) {
+	rt := defaultRuntime(t)
+
+	rt.setTargetInvalidBlockNumber(testdata.TargetBlockNumber)
+	rt.setupMaliciousValidator()
+	rt.setupHonestChallenger()
+
+	// bind contracts
+	rt.bindChallengeContracts()
+
+	// create challenge
+	rt.setupChallenge()
+
+interaction:
+	for {
+		status, err := rt.colosseumContract.GetStatus(nil, rt.outputIndex)
+		require.NoError(rt.t, err)
+
+		switch status {
+		case chal.StatusAsserterTurn:
+			// do nothing to trigger asserter timeout
+			rt.miner.ActEmptyBlock(rt.t)
+		case chal.StatusChallengerTurn:
+			// call bisect by challenger
+			rt.txHash = rt.challenger.ActBisect(rt.t, rt.outputIndex)
+			rt.miner.includeL1Block(rt.t, rt.challenger.address)
+		case chal.StatusAsserterTimeout:
+			rt.txHash = rt.challenger.ActProveFault(rt.t, rt.outputIndex, false)
```

Comment:
This should be `true`, which make test succeeds
```suggestion
			rt.txHash = rt.challenger.ActProveFault(rt.t, rt.outputIndex, true)
```

Reply:
@0xHansLee  ur right. thx!